Purchases/Public/RCPurchases.m

```diff
@@ -673,12 +675,30 @@ - (void)checkTrialOrIntroductoryPriceEligibility:(NSArray<NSString *> *)productI
                                  completionBlock:(RCReceiveIntroEligibilityBlock)receiveEligibility
 {
     [self receiptData:^(NSData * _Nonnull data) {
-        [self.backend getIntroEligibilityForAppUserID:self.appUserID
-                                          receiptData:data
-                                   productIdentifiers:productIdentifiers
-                                           completion:^(NSDictionary<NSString *,RCIntroEligibility *> * _Nonnull result) {
-                                               CALL_IF_SET_ON_MAIN_THREAD(receiveEligibility, result);
-                                           }];
+        LocalReceiptParser *receiptParser = [[LocalReceiptParser alloc] init];
```

Comment:
this is a proof of concept, we'd want to move this logic somewhere nicer and use DI for the LocalReceiptParser before shipping